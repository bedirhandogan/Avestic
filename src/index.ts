export { default as Avestic } from './avestic.vue'
