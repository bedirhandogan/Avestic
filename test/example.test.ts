import { describe, expect, it } from 'vitest'
import { mount } from '@vue/test-utils'
import { Avestic } from '../src'

describe('avestic', () => {
  it('disabled', () => {
    const wrapper = mount(Avestic, {
      props: {
        disabled: true,
      },
    })

    expect(wrapper.html()).toMatchSnapshot()
  })
})
