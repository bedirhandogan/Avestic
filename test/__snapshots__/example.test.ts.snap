// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`av > disabled 1`] = `"<div disabled="true"><img src="" alt=""></div>"`;

exports[`avestic > disabled 1`] = `"<div disabled="true"><img src="" alt=""></div>"`;

exports[`button > disabled 1`] = `"<div disabled="true"><img src="" alt=""></div>"`;
